plugins/tap.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: tap
+spec:
+  version: v0.1.1
+  homepage: https://soluble-ai.github.io/kubetap/
+  shortDescription: "interactively proxy Kubernetes Services with ease"
+  description: |
+    Interactively proxy Kubernetes Services by deploying
+    intercepting proxy sidecars.
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/soluble-ai/kubetap/releases/download/v0.1.1/kubetap_0.1.1_Darwin_x86_64.tar.gz
+    sha256: 62a7832c7edcff145ec1dc22a04304ac921fc2c9bf723b21142a04d1e6a65a8c
+    bin: kubectl-tap
+    files:
+    - from: "*"
+      to: "."
```

Comment:
this is the default and not needed, you can remove highlighted portion.

Reply:
done.